jest.config.integration.js

```diff
@@ -1,4 +1,4 @@
 const config = require('./jest.config');
-config.testRegex = '.itest\\.js$'; //Overriding testRegex option
+config.testRegex = '\\.itest\\.js$'; //Overriding testRegex option
```

Comment:
Is it possible to just use an inline regex for this? `/\.itest\.js$/` might look _slightly_ cleaner

Reply:
Unfortunately, it is not. Jest wants a string or array of strings. It won't accept a raw regex.